src/platform/lunarlake/include/platform/lib/dma.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+#ifdef __SOF_LIB_DMA_H__
+
+#ifndef __PLATFORM_LIB_DMA_H__
+#define __PLATFORM_LIB_DMA_H__
+
+/* max number of supported DMA channels */
+#define PLATFORM_MAX_DMA_CHAN	9
+
+/* number of supported DMACs */
+#define PLATFORM_NUM_DMACS	6
```

Comment:
very odd, we have 9 and 11 above.

Reply:
Most of the definitions from /platform/... directory is not used for IPC4, however lack of this definition causes errors. This should be cleaned up for LNL and MTL as well. In separate PR I think.
But fixed for now.